create-frozen-vm.sh

```diff
@@ -22,17 +26,22 @@ while [[ $# -gt 0 ]]; do
             enable_frozenvm_each_host=true
             shift # Move to the next argument
             ;;
+        --os)
+            # Set the os of frozen vm
+            os=$2
+            shift 2 # Move to the next argument
+            ;;
         *)
```

Comment:
We need to add some help msg to tell people the full list of the supported "os" arguments.